Assets/Scripts/GUI/Slider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Slider : MonoBehaviour
```

Comment:
Rename to ChainSlider or something like that(Same name as UnityEngine.UI.Slider)